test/functional/api/highlight_spec.lua

```diff
@@ -337,4 +337,10 @@ describe("API: set highlight", function()
       exec_capture('highlight Test_hl3'))
 
   end)
+
+  it ("correctly sets 'Normal' internal properties", function()
```

Comment:
Hardest part of the test was naming it and while the issue is listed with get_hl_by_name, the cause is closer to set_hl, so I stuck it here.

Reply:
thank you for trying to find a meaningful name!